openmls/src/group/core_group/mod.rs

```diff
@@ -31,20 +31,29 @@ mod test_past_secrets;
 mod test_proposals;
 
 #[cfg(test)]
-use super::errors::CreateGroupContextExtProposalError;
-use super::public_group::PublicGroup;
-use crate::binary_tree::array_representation::TreeSize;
+use std::io::{Error, Read, Write};
```

Comment:
This was automatically cleaned-up using `rustfmt --config imports_granularity=Crate,group_imports=StdExternalCrate openmls/src/group/tests/test_external_commit_validation.rs`. Looking through the codebase, I think we pretty much agree on this style, thus, I configured my IDE to do this in changed files. I could also make a PR that does all this as a one-time pill but refrained from it to not introduce conflicts.

Reply:
I hoped it would create less churn, but yeah, reformating "bit-by-bit" in PRs isn't the best idea. Sadly, the rustfmt options we would need to do it once and enforce in CI are still nightly-only :-(